dfusion_rust_core/src/database/graph_reader.rs

```diff
@@ -123,7 +123,7 @@ impl DbInterface for GraphReader {
                     ),
                     EntityFilter::Equal(
                         "accountId".to_string(),
-                        (reserved_account_id as u16).to_value(),
+                        H160::from_low_u64_be(reserved_account_id as _).to_value(),
```

Comment:
Why is this `as _` needed?

Reply:
Its coming from the crate import in `listener/src/main`

```
use graph::prelude::{SubgraphRegistrar as _, *};
```

Reply:
reserved_account_id needs to be cast from a `usize` to a `u64`. Since the function name is very explicit about it taking a u64, I thought that the `as _` was OK to use. It basically tells the compiler "cast to what is needed".

Reply:
We can rewrite it as `as u64` if you feel strongly about it. Its just some type-inference magic.

Reply:
Whoops... ignore my comment then.